x-pack/plugins/enterprise_search/public/applications/workplace_search/views/role_mappings/role_mappings_logic.ts

```diff
@@ -280,12 +280,9 @@ export const RoleMappingsLogic = kea<MakeLogicType<RoleMappingsValues, RoleMappi
         }
       }
     },
-    handleDeleteMapping: async () => {
-      const { roleMapping } = values;
-      if (!roleMapping) return;
-
+    handleDeleteMapping: async ({ roleId }) => {
       const { http } = HttpLogic.values;
-      const route = `/api/workplace_search/org/role_mappings/${roleMapping.id}`;
```

Comment:
I propose to keep the naming.
roleId -> roleMappingId

When I see `roleId`, I think of 'admin' or 'user' and not a random string of `roleMappingId`

Reply:
https://github.com/elastic/kibana/pull/101198/commits/13472709b9d8b627aa68f33331f00eec5a675cd7